### PR TITLE
added filter choice in write_center

### PR DESCRIPTION
**Pull request by vdeandrade**

### CODE REVIEW

File: tomopy/recon/rotation.py [hunk 1]
```diff
@@ -155,7 +155,9 @@ def _adjust_hist_limits(tomo_ind, theta, mask, sinogram_order):
     rec = recon(tomo_ind, 
                 theta,
                 sinogram_order=sinogram_order, 
-                algorithm='gridrec')
+                algorithm=algorithm,
```
Comment (dgursoy) [inline tomopy/recon/rotation.py]: I think this should be ```algorithm='gridrec'```.

File: tomopy/recon/rotation.py [hunk 2]
```diff
@@ -191,7 +193,7 @@ def _find_center_cost(
     center = np.array(center, dtype='float32')
     rec = recon(
         tomo_ind, theta, center,
-        sinogram_order=sinogram_order, algorithm='gridrec')
+        sinogram_order=sinogram_order, algorithm=algorithm, filter_name=filter_name)
```
Comment (dgursoy) [inline tomopy/recon/rotation.py]: Sane here: ```algorithm='gridrec'```.

Comment (decarlof) [reply tomopy/recon/rotation.py]: @dgursoy  this was a change I did with @vdeandrade long time ago .... I think is better to have a default as you did. @vdeandrade: any comment?

**Comment by decarlof** [comment]
closed. Changes will be included in a new pull request

**Pull request closed without merging**